### PR TITLE
v1.6 release note

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

File: docs/en_US/Release.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+### NAS Updates
+* Use OrderedDict for LayerChoice
+* Prettify the format of export
+* Replace layer choice with selected module after applied fixed architecture
+
+### Model Compression Updates
+* Model compression PyTorch 1.4 support
+
+### Training Service Updates
+* support pai multiphase
```
Comment (squirrelsc) [inline docs/en_US/Release.md:35]: As we've removed multiphase, can we hide this?

File: docs/en_US/Release.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+* Model compression PyTorch 1.4 support
+
+### Training Service Updates
+* support pai multiphase  
+* update pai yaml merge logic
+* support windows as remote machine in remote mode [Remote Mode](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/RemoteMachineMode.md#windows)
+
+### Bug Fix
+* fix dev install
+* fix windows dev intall
```
Comment (squirrelsc) [inline docs/en_US/Release.md:41]: `support windows dev install`

File: docs/en_US/Release.md (hunk 1 above)
Comment (SparkSnail) [reply docs/en_US/Release.md]: fixed.

File: docs/en_US/Release.md (hunk 2 above)
Comment (SparkSnail) [reply docs/en_US/Release.md]: fixed.

**Pull request merged**